includes/polling/applications/ups-nut.inc.php

```diff
@@ -27,12 +33,11 @@
 
 echo ' '.$name;
 
-list ($charge, $bat_low, $remaining, $bat_volt, $model, $serial, $input_vol, $line_vol, $load) = explode("\n", $ups_nut);
+list ($charge, $remaining, $bat_volt, $model, $serial, $input_volt, $line_volt, $load) = explode("\n", $ups_nut);
 
 $rrd_name = array('app', $name, $app_id);
 $rrd_def = array(
     'DS:charge:GAUGE:600:0:100',
-    'DS:battery_low:GAUGE:600:0:100',
```

Comment:
if it is a older rrd with a number of x DS any change to that number of DS would render the old rrd unusable that meaning that you would have to start with a fresh rrd loosing all data before that point